main.go

```diff
@@ -70,6 +75,42 @@ func main() {
 			zone = ""
 		}
 	}
+
+	// Generate deployment info from metadata server or from command-line
+	// arguments, with the latter taking preference.
+	var deploymentInfo map[string]string
+	if *includeDeploymentInfo {
+		dType := getDeploymentType()
+		switch dType {
+		case deploymentTypeGKE:
+			cluster := *gkeClusterName
+			if cluster == "" {
+				cluster = getClusterName()
+			}
+			pod := *gkePodName
+			if pod == "" {
+				pod = getPodName()
+			}
+			deploymentInfo = map[string]string{
+				"GKE-CLUSTER":   cluster,
+				"GCP-ZONE":      zone,
+				"INSTANCE-IP":   ip,
+				"GKE-POD":       pod,
+				"GKE-NAMESPACE": *gkeNamespace,
+			}
+		case deploymentTypeGCE:
+			vmName := *gcpVM
+			if vmName == "" {
+				vmName = getVMName()
+			}
+			deploymentInfo = map[string]string{
+				"GCP-VM":      vmName,
```

Comment:
nit: can this be GCE-VM to be inline with other places where we use GKE-*

Reply:
Thanks for the catch. Fixed it.